stdlib/builtins.pyi

```diff
@@ -111,32 +112,32 @@ class object:
     def __dir__(self) -> Iterable[str]: ...
     def __init_subclass__(cls) -> None: ...
 
-class staticmethod(Generic[_R_co]):
+class staticmethod(Generic[_P, _R_co]):
     @property
-    def __func__(self) -> Callable[..., _R_co]: ...
+    def __func__(self) -> Callable[_P, _R_co]: ...
     @property
     def __isabstractmethod__(self) -> bool: ...
-    def __init__(self: staticmethod[_R_co], __f: Callable[..., _R_co]) -> None: ...
-    def __get__(self, __instance: _T, __owner: type[_T] | None = None) -> Callable[..., _R_co]: ...
+    def __init__(self, __f: Callable[_P, _R_co]) -> None: ...
+    def __get__(self, __instance: _T, __owner: type[_T] | None = None) -> Callable[_P, _R_co]: ...
     if sys.version_info >= (3, 10):
         __name__: str
         __qualname__: str
         @property
-        def __wrapped__(self) -> Callable[..., _R_co]: ...
-        def __call__(self, *args: Any, **kwargs: Any) -> _R_co: ...
+        def __wrapped__(self) -> Callable[_P, _R_co]: ...
+        def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _R_co: ...
 
-class classmethod(Generic[_R_co]):
+class classmethod(Generic[_P, _R_co, _T]):
```

Comment:
Maybe it better put `_T` first to reflect a typical method signature?
```
def meth(cls:_T, *args: _P.args, **kwargs: _P.kwargs) -> _R_co: ...
```

Reply:
Done!